### ISSUE TITLE
Tutorials on first day too long

### ISSUE THREAD

**Issue by fransua** [Pfern/3DAROC#1]
The parsing of reads, and perhaps even part of the explanation of the mapping should be moved to the next day.

**Comment by Pfern** [comment]
In the next edition, you mean!

**Comment by fransua** [comment]
... yes :)
my secret plan, is to use these issues to improve the course on the next edition. 
If you have any other kind of suggestion, perhaps you could also put them here. 
Otherwise, it will be impossible to remember what went wrong, and what could be improved.

For the next edition, we could then, for example, adjust this timing problem, and if it works better, close the issue.

btw, do you plan to have one repo by course edition? I think it would be nice to have a single repo for this course, and use TAGs to label the different editions.

**Comment by Pfern** [comment]
Yes, but TAGs look confusing with a course that evolves, which is a the case.
What I am doing nor is that I keep different editions in different branches, when there is a standing ser of training contents. 
Another thing that I am preparing to do is to add short video clips. The SONY Alpha 6000 is for that. I also bought a gib crane and a video illuminator for that purpose.

**Comment by Pfern** [comment]
We could even do what I am starting with others. I finish a course, make a polishing intervention and freeze that content as documentation. Then I open a new repo and using the previous content and the new (improvement) comments go there.
